### PR TITLE
67 be setting up auth

### LINKED ISSUE #67: BE: Setting up auth
Install `pip install django-allauth dj-rest-auth djangorestframework`

Read more here:
[django-all-auth](https://docs.allauth.org/en/latest/)
[dj-rest-auth](https://dj-rest-auth.readthedocs.io/en/latest/) (will plan to use JWT/SimpleToken (django)/sessions))

### PR DESCRIPTION
**Pull request by ewandeyb**
sorry in advance, pero working na ni na user model, wala pa na reference sa address table tho

[dj-rest-auth](https://dj-rest-auth.readthedocs.io/en/latest/installation.html)
[all-auth](https://docs.allauth.org/en/latest/)

### CODE REVIEW

File: backend/kitschy_api/managers.py [hunk 1]
```diff
@@ -13,17 +13,17 @@ def create_user(self, email, password, **extra_fields):
         user.set_password(password)
         user.save()
         return user
-    
+
     def create_superuser(self, email, password, **extra_fields):
-        """"
+        """ "
```
Comment (44mira) [inline backend/kitschy_api/managers.py]: Typo?

**Review by 44mira — CHANGES_REQUESTED** [review]
Fix typo and guarantee that `./manage.py runserver` does not break. 

Attach a screenshot of the working API `/docs` endpoint.

File: backend/kitschy_api/managers.py (hunk 1 above)
Comment (ewandeyb) [reply backend/kitschy_api/managers.py]: ay dle bai

Comment (ewandeyb) [reply backend/kitschy_api/managers.py]: SS of docs/ path
image

**Review by 44mira — APPROVED** [review]
LGTM.

**Pull request merged**